playkit/src/main/java/com/kaltura/playkit/player/PKExternalSubtitle.java

```diff
@@ -14,6 +15,11 @@
     private String id;
     private String mimeType;
     private int selectionFlags = Consts.TRACK_UNSELECTED_FLAG;
+    /**
+     * Indicates the track contains subtitles. This flag may be set on video tracks to indicate the
+     * presence of burned in subtitles.
+     */
+    private int roleFlag = C.ROLE_FLAG_SUBTITLE;
```

Comment:
Where is this field used?

Reply:
Fixed. It is being used in `ExoPlayerWrapper` in `buildExternalSubtitleSource`